TYPO3.Neos/Classes/TYPO3/Neos/Controller/Backend/MenuHelper.php

```diff
@@ -64,8 +64,10 @@ public function buildSiteList(ControllerContext $controllerContext)
             $active = false;
             /** @var $site \TYPO3\Neos\Domain\Model\Site */
             if ($site->hasActiveDomains()) {
-                $hostname = $site->getFirstActiveDomain()->getHostPattern();
-                $active = $hostname === $requestUriHost;
+                $activeDomains = $site->getActiveDomains()->map(function ($domain) {
```

Comment:
What I meant is, that after this call `$activeDomains` is an array of strings representing host patterns of active domains, so the name `$activeDomain` is a bit misleading.. Not a biggy though